ui-tests/cypress/support/controllers/restAPI/SeoPagesController.js

```diff
@@ -17,4 +17,14 @@ export default class SeoPagesController extends AbstractController {
     });
   }
 
+  setPageJoinGroups(page, ownerGroup, newJoinGroups) {
```

Comment:
You're setting both _joinGroups_ and _ownerGroup_; rename the method accordingly